README _En.md

```diff
@@ -0,0 +1,44 @@
+# Bitcoin & Lightning Network `Francophone` Ressources HUB
```

Comment:
Cool stuff @richarddushime .

I think you should remove the white space in the file name. It would be better to have `README-fr` or `README-en`

Reply:
Sure Thanks